contributors/devel/release.md

```diff
@@ -0,0 +1,231 @@
+# Targeting Features, Issues and PRs to Release Milestones
+
+This document is focused on kubernetes developers and contributors
+who need to create a feature, issue, or pull request which targets a specific
+release milestone.
+
+-   [Definitions](#definitions)
+-   [The Release Cycle](#the-release-cycle)
+-   [Removal Of Items From The Milestone](#removal-of-items-from-the-milestone)
+-   [Adding An Item To The Milestone](#adding-an-item-to-the-milestone)
+    -   [Milestone Maintainers](#milestone-maintainers)
+    -   [Feature additions](#feature-additions)
+    -   [Issue additions](#issue-additions)
+    -   [PR Additions](#pr-additions)
+    -   [Other Required Labels](#other-required-labels)
+        -   [SIG Owner Label](#sig-owner-label)
+        -   [Priority Label](#priority-label)
+        -   [Issue Kind Label](#issue-kind-label)
+
+The process for shepherding features, issues, and pull requests
+into a kubernetes release spans multiple stakeholders:
+* the feature, issue, or pull request owner
+* SIG leadership
+* the release team
+Information on workflows and interactions are described below.
+
+As the owner of a feature, issue, or pull request (PR), it is your
+responsibility to ensure release milestone requirements are met.
+Automation and the release team will be in contact with you if
+updates are required, but inaction can result in your work being
+removed from the milestone.  Additional requirements exist when the
+target milestone is a prior release (see [cherry pick
+process](cherry-picks.md) for more information).
+
+In the past there was a requirement for a milestone targeted pull
+request to have an associated GitHub issue opened, but this is no
+longer the case.  Features are effectively GitHub issues (or KEPs)
+which lead to subsequent PRs.  The terms feature, issue, and PR may
+be used interchangeably below, but the general labeling process
+should be consistent across artifact types.
```

Comment:
This is really confusing to me. It seems to say that issues and PRs are the same, and I don't think that's the intent here. I think the explanation below is very clear, and perhaps we can take out the part of this sentence before the comma.

Reply:
Yes.  I'd started to turn the issues.md doc into an issues and PRs doc, but really over the last couple weeks have realized this is a release process doc.  Once I added some info on features, it muddles this sentence.  The aim is the part after the comma...consistent treatment of artifacts associated with the release for simplest user experience.  Workflows are different though for the different artifact types and are given separate sections as of this draft.